lib/InternalRequest.php

```diff
@@ -7,28 +7,52 @@
 class InternalRequest {
     use Struct;
 
+    /** @var Client */
     public $client;
+    /** @var Vhost */
     public $vhost;
+    /** @var callable|null */
     public $preAppResponder;
+    /** @var Response */
     public $response;
+    /** @var \Generator */
     public $responseWriter;
+    /** @var array */
     public $badFilterKeys = [];
+    /** @var boolean */
     public $filterErrorFlag;
+    /** @var boolean */
     public $isServerStopping;
+    /** @var string */
     public $streamId;
```

Comment:
I somehow had in mind that we discussed that this needs to be a string so it can "never" overflow?